opentreemap/works_management/models.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+from django.contrib.gis.db import models
+from django.utils.translation import ugettext_lazy as _
+from django.utils import timezone
+
+from treemap.models import MapFeature, User
+from treemap.audit import Auditable
+from treemap.udf import UDFModel
+from treemap.instance import Instance
+
+
+class Team(models.Model):
+    instance = models.ForeignKey(Instance)
+    name = models.CharField(max_length=255)
+
+
+class WorkOrder(models.Model, Auditable):
+    instance = models.ForeignKey(Instance)
+    name = models.CharField(max_length=255)
```

Comment:
It should be non-nullable by default, right? But I'll set `null=False` and `blank=False` just to be safe.